prefs/HBListController.m

```diff
@@ -61,6 +61,69 @@ - (NSMutableArray *)loadSpecifiersFromPlistName:(NSString *)plistName target:(PS
 	return [self _hb_configureSpecifiers:specifiers];
 }
 
+- (NSInteger)imageSystemWeight:(id)weight {
```

Comment:
Hmm, let’s make a new `SFSymbols.h` and add these in there as static funcs, like:

```objc
static UIImageSymbolWeight systemImageWeight(id weight) {
	…
}
```

Then change it to a lookup table:

```objc
NSDictionary <NSString *, NSNumber *> *values = @{
	@"UIImageSymbolWeightUltraLight": @(UIImageSymbolWeightUltraLight),
	@"ultraLight": @(UIImageSymbolWeightUltraLight),
	// etc etc
};

NSNumber *value = values[weight];
if (value != nil) {
	return (UIImageSymbolWeight)value.integerValue;
}
return UIImageSymbolWeightMedium;
```